### PR TITLE
[ENH] Added `Assessment Tool` annoatatoins to the JSON output file

### LINKED ISSUE #585: Make sure assessment tool annotations make it to the output .json
With #285 completed, we need to also make sure that the assessment tool annotations successfully make it into the .json output according to our data dictionary specification: https://neurobagel.org/dictionaries/#assessment-tool

- [x] add assessment tool mappings to the JSONoutput getter in the store
- [x] ensure that the output .json includes the correctly formatted information
- [x] test the new functionality, including with an e2e test

### PR DESCRIPTION
**Pull request by rmanaem**
Closes #585 

## Checklist

- [x] PR has an interpretable title with a prefix (`[ENH]`, `[FIX]`, `[REF]`, `[TST]`, `[CI]`, `[MNT]`, `[INF]`, `[MODEL]`, `[DOC]`) _(see https://neurobagel.org/contributing/pull_requests for more info)_
- [x] PR links to GitHub issue with mention `Closes #XXXX`
- [ ] Tests pass
- [ ] Checks pass

For new features:
- [ ] Tests have been added

For bug fixes:
- [ ] There is at least one test that would fail under the original bug conditions.

### CODE REVIEW

File: components/annot-tool-groups.vue [hunk 1]
```diff
@@ -12,7 +12,7 @@
                     :title="label">
                     <annot-single-tool
                         :name="identifier"
-                        :uniqueColumnValues="provideUniqueValues(identifier)"
+                        :unique-column-values="provideUniqueValues(identifier)"
```
Comment (surchs) [inline components/annot-tool-groups.vue:15]: :eyes:

is this eslint?

File: cypress/unit/store-getter-getAssessmentToolJSONOutput.cy.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        }
+
+    }
+};
+
+describe("getAssessmentToolJSONOutput", () => {
+
+    it("Make sure Assessment tool json output is schema compliant", () => {
+
+        // Act - Get formatted json output data for discrete value column
```
Comment (surchs) [inline cypress/unit/store-getter-getAssessmentToolJSONOutput.cy.js:34]: I'm OK with adding these flags, but then we should also add it for "Assert" below. That's the main benefit to me: you see where the asserts begin

**Review by surchs — APPROVED** [review]
small edits, then :cook:

File: components/annot-tool-groups.vue (hunk 1 above)
Comment (rmanaem) [reply components/annot-tool-groups.vue:15]: Yea

**Pull request merged**